crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -223,6 +228,7 @@ def save_and_restore
         flash[:info] = t("installer.upgrades.restore.multiple_restore")
         true
       else
+        Crowbar::Backup::Restore.purge
```

Comment:
I'm a bit confused here: we are purging the restore steps file with this, but the test in the `if` condition is checking if it exists. Can't this cause a problem? Or is the restore steps file remove on failure anyway?